frontend/src/features/admin-form/create/builder-and-design/components/BuilderAndDesignDrawer/EditFieldDrawer/EditRadio.tsx

```diff
@@ -0,0 +1,200 @@
+      required: field.required,
+      othersRadioButton: field.othersRadioButton,
+    },
+  })
+
+  const watchedInputs = watch()
+
+  useDebounce(
+    () => {
+      if (!watchedInputs) return
+      return updateActiveField(transformToFormField(watchedInputs))
+    },
+    300,
+    [
+      transformRadioOpts,
+      // Required destructure to prevent debounce firing infinitely.
+      watchedInputs.description,
+      watchedInputs.fieldOptions,
+      watchedInputs.required,
+      watchedInputs.title,
+      watchedInputs.othersRadioButton,
+    ],
+  )
+
+  const { mutateFormField } = useMutateFormFields()
+
+  const saveButtonText = useMemo(
+    () => (isPendingFormField(field) ? 'Create' : 'Save'),
+    [field],
+  )
+
+  const isSaveDisabled = useMemo(
+    () => isDirty || isPendingFormField(field),
+    [field, isDirty],
+  )
+
+  const handleUpdateField = handleSubmit((inputs) => {
+    const updatedField = extend({}, field, transformToFormField(inputs))
+    return mutateFormField.mutate(updatedField, {
+      onSuccess: () => {
+        reset(inputs)
+      },
+    })
+  })
+
+  const [tabIndex, setTabIndex] = useState(0)
+
+  // Effect to move to second tab to show error message.
+  useEffect(() => {
+    if (isSubmitting && tabIndex === 0) {
+      setTabIndex(1)
+    }
+  }, [isSubmitting, tabIndex])
+
+  return (
+    <DrawerContentContainer>
+      <Stack spacing="2rem" divider={<Divider />}>
+        <FormControl
+          isRequired
+          isReadOnly={mutateFormField.isLoading}
+          isInvalid={!!errors.title}
+        >
+          <FormLabel>Question</FormLabel>
+          <Input autoFocus {...register('title', requiredValidationRule)} />
+          <FormErrorMessage>{errors?.title?.message}</FormErrorMessage>
+        </FormControl>
+        <FormControl
+          isRequired
+          isReadOnly={mutateFormField.isLoading}
+          isInvalid={!!errors.description}
+        >
+          <FormLabel>Description</FormLabel>
+          <Textarea {...register('description')} />
+          <FormErrorMessage>{errors?.description?.message}</FormErrorMessage>
+        </FormControl>
+        <FormControl
+          isRequired
+          isReadOnly={mutateFormField.isLoading}
+          isInvalid={!!errors.fieldOptions}
+        >
+          <FormLabel>Options</FormLabel>
+          <Textarea {...register('fieldOptions', fieldOptionsValidationRule)} />
+          <FormErrorMessage>{errors?.fieldOptions?.message}</FormErrorMessage>
+        </FormControl>
+        <Toggle
+          isLoading={mutateFormField.isLoading}
+          label="Others"
+          {...register('othersRadioButton')}
+        />
+        <Toggle
+          isLoading={mutateFormField.isLoading}
+          label="Required"
+          {...register('required')}
+        />
+      </Stack>
+
+      <FormFieldDrawerActions
+        isLoading={mutateFormField.isLoading}
+        isDirty={isSaveDisabled}
```

Comment:
This will probably conflict with [Anty's latest PR](https://github.com/opengovsg/FormSG/pull/3547/files#diff-1392ff3e5d630aeb498d11509b21b63a04a4dfa412ba34e91ad70a6118174801R127)